main.tf

```diff
@@ -11,37 +12,7 @@
   tags                           = var.tags
   vpc_config                     = var.vpc_config
   layers                         = var.layers
-
- # additions from RV standard
-  name                = var.function_name
-  project             = var.project_name
-  service             = var.service
-  owner               = var.owner # || vertical
-  team_name           = var.team_name
-  resource_allocation = var.resource_allocation
-
-  # Additions from old lambda sub-module
-  /*
-  dynamic "environment" {
-    for_each = length(var.environment) < 1 ? [] : [var.environment]
-    content {
-      variables = environment.value.variables
-    }
-  }
-
-  function_name                  = var.function_name
-  memory_size                    = var.memory_size
-  role                           = aws_iam_role.lambda.arn
-  source_code_hash               = filebase64sha256(var.filename)
-
-  dynamic "vpc_config" {
-    for_each = length(var.vpc_config) < 1 ? [] : [var.vpc_config]
-    content {
-      security_group_ids = vpc_config.value.security_group_ids
-      subnet_ids         = vpc_config.value.subnet_ids
-    }
-  }
-  */
+  resource_allocation            = var.resource_allocation
 
   # bonus points
   #create_in_vpc     = var.create_in_vpc
```

Comment:
Do we have migration plans or a feature parity roadmap for folks migrating from the old module?